### ISSUE TITLE
New Comm Card Proposal: Download Trend

### ISSUE THREAD

**Issue by AARON-CLARK** [pharmaR/riskassessment#438]
I want to propose a new card that fits a linear model to the monthly data produced by `generate_comm_data()` with the goal of reporting on the slope coefficient.  For example, below you can see that the `stringr` package has been increasing in downloads over the last two years. Let's say we generate a linear model on this data and the slope is 27k. In this context, we could highlight two things:

* First, the the trend is positive/ upwards!
* Second, that the average increase in pkg downloads increases by 27k per month.

I think those are really handy statistics to know! Maybe we add a button to overlay the linear model on the plot? We probably want to limit the model fit to the last two years to keep the stats recent... but I guess we could always empower the user to choose the timeframe. On the lower bound, we shouldn't fit a model unless a minimum of 12 months exists.

Perhaps if we implement and test some scenarios, this could be an assessment we propose to `riskmetric`?

Thoughts?

This current Community Usage Tab:

![image](https://user-images.githubusercontent.com/17878953/220915732-10577a8b-65c9-4ad0-8acc-739d6273722b.png)

**Comment by Eduardodudu** [comment]
Hi!

I have added a first approach for the card, for now, the timeframe is strict for 12 months and the card is not shown when it does not fit the minimum of 12 months.


![Image](https://github.com/pharmaR/riskassessment/assets/8216229/7d0a17c2-a4da-46eb-a078-996afab2106e)

![Image](https://github.com/pharmaR/riskassessment/assets/8216229/ce265071-cf5c-4354-9b78-de3f122a22ec)


For the plot, I added the linear trend of 2 years, since plotly was being used straight without ggplot, I couldn't replicate geom_smooth and had to create the lm + predict approach:



![Image](https://github.com/pharmaR/riskassessment/assets/8216229/6a591da9-1521-4031-9d18-e372b4d83e00)


I also added legends at the bottom

Let me know what you think about it, there is a draft pull request of this task.
